### ISSUE TITLE
Useless information

### ISSUE THREAD

**Issue by wahboh** [dotnet/docs#5078]
Assume that I want to write a handler for this event. Where do I find the interface description? 
The new documnetation is all to often totally useless lacking substantial information and examples.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f2c2a4b4-423c-e2b5-8dec-bd8c0f046b7c
* Version Independent ID: 30a347cb-8b49-334e-1b3d-192e57bffe5b
* Content: [FrameworkElement.DataContextChanged Event (System.Windows)](https://docs.microsoft.com/en-us/dotnet/api/system.windows.frameworkelement.datacontextchanged?view=netframework-4.7.1)
* Content Source: [xml/System.Windows/FrameworkElement.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows/FrameworkElement.xml)
* Service: **unspecified**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

**Comment by BillWagner** [comment]
Removing the external label: This content should link to the `System.Windows.DependencyPropertyChangedEventHandler` type, which has all the information needed.

**Comment by svick** [comment]
@BillWagner Shouldn't all pages for events link to their event handler type? I think that would be an "external" issue.

**Comment by BillWagner** [comment]
@svick Yes, in general. Some use a type of `EventHandler` which has no arguments, so that's not super useful.

@dend What are your thoughts?

**Comment by wahboh** [comment]
Dear All,
Please do not underestimate the power of examples. Many doc pages are just the spec, but I can vouch  from loong experience that what most pro programmers really need, are small examples on howto use. Can oneliners, or very brief, but it says much more than just the spec.

**Comment by BillWagner** [comment]
@wahboh I agree. Samples are great. What we need to do is make the samples easy to find, which means getting them on the right pages, and then linking to them from other relevant pages. Which do you  think would be easier to find: the event, or the event handler type?

**Comment by dend** [comment]
So to distill the problem, I think there are two asks:

1. Linking to the base event handler (in the case above, `System.Windows.DependencyPropertyChangedEventHandler`) from the event page.
2. Adding more examples for events.

For (1), am I interpreting the request correctly?

@BillWagner @wahboh

**Comment by wahboh** [comment]
@BillWagner @dend 
I have no strong opinion about Bill's #1 above. Generally when I as a developer search for info, I look at the immediate info. I do not very often trace back to some base class. On the other hand I can agree that info should be found where the function (etc) is declared, i e probably in the base class.
I leave it to you pros to put yourself in the situation of a confused (and maybe a bit lazy) developer who need instant (!) information (with an example) on the immediate (!) problem. How many steps will she/he trace the stack backwards?
